Modules/System/AL Rest Client/app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "812b339d-a9db-4a6e-84e4-fe35cbef0c44",
+  "name": "AL Rest Client",
```

Comment:
No need to add `AL`, just `Rest Client` is good.

Reply:
Changed to "Rest Client".